ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -718,35 +718,12 @@ def test_paddle_take_along_axis(
     )
 
 
-# rot90
-@handle_frontend_test(
-    fn_tree="paddle.rot90",
-    dtype_m_k_axes=_get_dtype_values_k_axes_for_rot90(
-        available_dtypes=helpers.get_dtypes(kind="valid"),
-        min_num_dims=1,
-        max_num_dims=5,
-        min_dim_size=1,
-        max_dim_size=10,
-    ),
-)
-def test_paddle_rot90(
-    *,
-    dtype_m_k_axes,
-    on_device,
-    fn_tree,
-    frontend,
-    backend_fw,
-    test_flags,
-):
-    input_dtype, m, k, axes = dtype_m_k_axes
+Solved from the merge of gather_nd and main branches.
     helpers.test_frontend_function(
         input_dtypes=input_dtype,
         backend_to_test=backend_fw,
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
-        x=m,
-        k=k,
-        axes=tuple(axes),
-    )
+Again, Solved from the merge of gather_nd and main branches.
```

Comment:
same for this comment

Reply:
Ok, no problem. I will take a looks at it. Thank you.